File: Unidad 2-Estructuras de Control/Ejercicio3.java
/*
Autor: Pedro Cauich
Entrada: día de la semana que se quiere
Salida: nombre del día
*/

import java.util.Scanner;

public class Main{
    public static void main(String arg[]) {

        // Definir variables [Principio]
        Scanner entrada = new Scanner(System.in);
        int dia = 0;
        String salida = "";
        boolean auxiliar = true;

        String nombreDias[] = new String[7];
        nombreDias[0] = "Lunes";
        nombreDias[1] = "Martes";
        nombreDias[2] = "Miércoles";
        nombreDias[3] = "Jueves";
        nombreDias[4] = "Viernes";
        nombreDias[5] = "Sábado";
        nombreDias[6] = "Domingo";

        dia = entrada.nextInt();


        // Condicional, selección del día, validación y salida[Procedimiento]
        if (dia > 0 && dia <= 7){
            salida = nombreDias[dia - 1];
        } else {
            auxiliar = false;
        }

        //Entrega de salidas 
        if (auxiliar == true){
            System.out.println("El día correspondiente al número es: " + salida);
        } else {
            System.out.println(auxiliar);
        }
    }
}

File: Unidad 2-Estructuras de Control/Ejercicio3.c
#include <stdio.h>

/* run this program using the console pauser or add your own getch, system("pause") or input loop */

int main(int argc, char** argv) {
	
	int num;
	
	scanf("%i", &num);
	
	switch(num)
	{
		case 1:
			printf("Lunes");
		break;
		case 2:
			printf("Martes");
		break;
		case 3:
			printf("Miércoles");
		break;
		case 4:
			printf("Jueves");
		break;
		case 5:
			printf("Viernes");
		break;
		case 6:
			printf("Sábado");
		break;
		case 7:
			printf("Domingo");
		break;
	}
	
	return 0;
}